examples/example3/example3_withmanufactured.ipynb

```diff
@@ -0,0 +1,679 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7cebdcf1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Atot = Parameter(\"Atot\", 1.0, vol_unit)\n",
+    "# Phosphorylation of Adephos at the PM\n",
+    "kkin = Parameter(\"kkin\", 50.0, 1/sec)\n",
+    "curRadius = 2  # first radius value to test\n",
+    "# vol to surface area ratio of the cell (overwritten for each cell size)\n",
+    "VolSA = Parameter(\"VolSA\", curRadius/3, um)\n",
+    "r1 = []\n",
+    "for PMsub in [\"PMleft\", \"PMright\", \"PMfront\", \"PMback\", \"PMbottom\", \"PMtop\"]:\n",
+    "    r1.append(Reaction(f\"r1_{PMsub}\", [], [\"Aphos\"], param_map={\"kon\": \"kkin\", \"Atot\": \"Atot\", \"VolSA\": \"VolSA\"},\n",
+    "              eqn_f_str=\"kon*VolSA*(Atot - Aphos)\", species_map={\"Aphos\": \"Aphos\"}, explicit_restriction_to_domain=PMsub))\n",
+    "# Dephosphorylation of Aphos in the cytosol\n",
+    "kp = Parameter(\"kp\", 10.0, 1/sec)\n",
+    "r2 = Reaction(\"r2\", [\"Aphos\"], [], param_map={\"kon\": \"kp\"},\n",
+    "              eqn_f_str=\"kp*Aphos\", species_map={\"Aphos\": \"Aphos\"})"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "430c8b6e",
+   "metadata": {},
+   "source": [
+    "Create parameter and reaction containers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "09079b17",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pc = ParameterContainer()\n",
+    "pc.add([Atot, kkin, VolSA, kp])\n",
+    "\n",
+    "rc = ReactionContainer()\n",
+    "rc.add([*r1, r2])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "48c25774",
+   "metadata": {},
+   "source": [
+    "## Create/load in mesh\n",
+    "\n",
+    "In SMART we have different levels of meshes. Here, for our first mesh, we specify a sphere of radius 1.\n",
+    "\n",
+    "$$\n",
+    "\\Omega: r \\in [0, 1] \\subset \\mathbb{R}^3\\\\\n",
+    "\\text{where} \\qquad r = \\sqrt{x^2 + y^2 + z^2}\n",
+    "$$\n",
+    "\n",
+    "which will serve as our parent mesh, giving the overall cell geometry.\n",
+    "\n",
+    "Different domains can be specified within this parent mesh by assigning marker values to cells (3D) or facets (2D) within the mesh. A subdomain within the parent mesh, defined by a region which shares the same marker value, is referred to as a child mesh.\n",
+    "\n",
+    "Here, we have two child meshes corresponding to the 2 compartments specified in the compartment container. As defined above, \"PM\" is a 2D compartment defined by facets with marker value 10 and \"Cyto\" is a 3D compartment defined by cells with marker value 1. These subdomains are defined by:\n",
+    "- $\\Omega_{Cyto}: r \\in [0, 1) \\subset \\mathbb{R}^3$\n",
+    "- $\\Gamma_{PM}: r=1 \\subset \\mathbb{R}^3$\n",
+    "\n",
+    "We generate the parent mesh with appropriate markers using gmsh in the function `mesh_tools.create_spheres`"
```

Comment:
Text out of date with actual code, uses box not sphere. Also remove any commented (unused code, such as create spheres in the code block below).